Item/Item.module.less

```diff
@@ -105,43 +107,95 @@
 		&.inline {
 			// overriding ui/Item's inline-block
 			display: inline-flex;
+			min-width: @agate-item-inline-min-width;
 			max-width: @agate-item-inline-max-width;
 
 			.itemContent {
 				max-width: @agate-item-inline-content-max-width;
 			}
 
+			&:not(.centered) {
+				.itemContent {
+					min-width: @agate-item-inline-notcentered-itemcontent-min-width;
```

Comment:
Where did you find this specification?

Reply:
This variable needed to be created in order to fix some layout bug for .itemContent when it is not centered.
I found it in GUI specs:
![item-width](https://user-images.githubusercontent.com/63335823/113124214-f231f800-921d-11eb-8ef3-680e440ac084.png)

Reply:
🤔 I think the guide shows the padding of the content, not the min-width. I don't think it's neccessary.

Reply:
This variable needs to be set in order to avoid a bug with inline Item. When there is a long content and a long label with position after, when hovering over the Item, the width of Item will grow.

Reply:
Have you tried with the recent code? I think it's not reproduced without this.

Reply:
For gallium, should we have `min-width` for `inline` or not?

Reply:
I think the below code instead of line 104 to 124 works fine for &.inline block. please check out and modify for sillicon skin.

```
		&.inline {
			// overriding ui/Item's inline-block
			display: inline-flex;
			max-width: @agate-item-inline-content-max-width;

			.itemContent {
				min-width: @agate-item-inline-min-width;
			}

		}
```

Reply:
For Gallium skin, the inline style issue will be taken care of in PLAT-139507(https://jira2.lgsvl.com/browse/PLAT-139507).